Library/Formula/qbittorrent.rb

```diff
@@ -0,0 +1,36 @@
+class Qbittorrent < Formula
+  desc "Qt based free and reliable P2P Bittorrent client"
+  homepage 'http://www.qbittorrent.org'
+  url 'https://github.com/qbittorrent/qBittorrent/archive/release-3.2.3.tar.gz'
+  sha256 "816d611a0af50c469f52946c94b88c003f908ee031ee0e55a45273e4c64897c5"
+  head 'https://github.com/qbittorrent/qBittorrent.git',
+    :branch => ENV['QBT_BRANCH'] || "master"
+
+  depends_on "pkg-config" => :build
+  depends_on "qt" => :build
+  depends_on "libtorrent-rasterbar" => :build
+  depends_on "boost" => :build
+  depends_on "gnu-sed" => :build
+
+  resource "geoIP" do
+    url "http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz"
```

Comment:
It's difficult, to be honest. Debian hack around this in their `geoip` package by doing this in the bootstrap:

```
mkdir -p $DIR/data
if [ ! -f $DIR/data/GeoIP.dat  ]; then
      curl http://geolite.maxmind.com/download/geoip/database/GeoLiteCountry/GeoIP.dat.gz | gzip -d > $DIR/data/GeoIP.dat
fi
```

Not ideal, really. My gut feeling would be for Homebrew to mirror semi-up-to-date GeoIP.dat "releases", but the attribution element of the license requirement makes that slightly tricky.